src/main/java/org/folio/rest/impl/AccountsBulkAPI.java

```diff
@@ -0,0 +1,94 @@
+package org.folio.rest.impl;
+
+import static io.vertx.core.Future.succeededFuture;
+
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.core.Response;
+
+import org.folio.rest.domain.Action;
+import org.folio.rest.exception.AccountNotFoundValidationException;
+import org.folio.rest.exception.FailedValidationException;
+import org.folio.rest.jaxrs.model.BulkCheckActionRequest;
+import org.folio.rest.jaxrs.model.BulkCheckActionResponse;
+import org.folio.rest.jaxrs.resource.AccountsBulk;
+import org.folio.rest.service.action.validation.ActionValidationService;
+import org.folio.rest.service.action.validation.DefaultActionValidationService;
+import org.folio.rest.utils.ActionResultAdapter;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+public class AccountsBulkAPI implements AccountsBulk {
+  private final static Logger logger = LoggerFactory.getLogger(AccountsBulkAPI.class);
```

Comment:
I don't think this is a compliance issue, otherwise it wouldn't compile at all. In JLS they say "If two or more (distinct) field modifiers appear in a field declaration, it is customary, though not required, that they appear in the order consistent with that shown above". But I totally agree that it's better to keep it clean. Changed here and also in AccountsAPI.